site/public/css/email-status.css

```diff
@@ -0,0 +1,40 @@
+.btn{
+    position: absolute;
+    min-width: 10%;
+    height: 5vh;
+    top: 0px;
+    right: 0px;
+}
+.button-container{
+    position: relative;
+    height: 5vh;
+    margin: auto;
+    display: inline-flexbox;
+}
+.expand{
+    display: none;
+    padding: 1vh;
+}
+.status{
+    list-style: none;
+    border: 1px solid black;
+}
+.status-error{
+    background-color: var(--danger-red);
+}
+.status-warning{
+    background-color: transparent;
+}
+.status-success{
+    background-color: var(--good-green);
+}
+.status-container{
+    margin-top: 1vh;
+    margin-bottom: 1vh;
+}
+div{
```

Comment:
Similar as above, please do not make such a sweeping CSS assignment on a specific page. It makes it harder for other developers as they may get confused (until looking here) as why divs on the email status page act differently than anywhere else. If you want to have `word-wrap: break-word` everywhere on this page, then use a div to wrap the page content that sets this on a unique classname or id.

Reply:
Renamed the class to status-message and edited the twig file accordingly.